js/DropZone/DropZone.js

```diff
@@ -89,7 +92,7 @@ export default class DropZone {
      */
     addFiles (files, meta = {}) {
         const { valid, text } = this.validator.validate(files, this.files.length, this.size);
-        let processedFiles = [];
+        const processedFiles = [];
```

Comment:
If processedFiles is a const now, why you don't put it in the line above with a comma?